src/internal_telemetry/allocations/mod.rs

```diff
@@ -55,8 +55,8 @@ impl GroupMemStats {
     pub fn new() -> Self {
         let mut mutex = THREAD_LOCAL_REFS.lock().unwrap();
         let stats_ref: &'static GroupMemStatsStorage = Box::leak(Box::new(GroupMemStatsStorage {
-            allocations: arr![AtomicU64::new(0) ; 128],
-            deallocations: arr![AtomicU64::new(0) ; 128],
+            allocations: arr![AtomicU64::new(0) ; 256],
```

Comment:
Nit-Use the constant here and below:
```suggestion
            allocations: arr![AtomicU64::new(0) ; NUM_GROUPS],
```

Reply:
i did look to use NUM_GROUPS instead of an integer but the arr macro expects integer literal. For this reason and my limited knowledge i decided to keep the change as minimal as possible with amending the existing integers.

Reply:
I see. We should be able to do `std::array::from_fn(|_| AtomicU64::new(0))` and delete the `arr_macro` dependency.

Reply:
> is it possible to accept the limited fix and look to refactor later?

Yes!